anvil/src/config.rs

```diff
@@ -172,7 +172,8 @@ Available Accounts
         );
         let balance = format_ether(self.genesis_balance);
         for (idx, wallet) in self.genesis_accounts.iter().enumerate() {
-            let _ = write!(config_string, "\n({idx}) {:?} ({balance} ETH)", wallet.address());
+            let _ = write!(config_string, "\n({idx}) {:?} ({balance} ETH)", to_checksum(&wallet.address(), self.chain_id.and_then(|id| Some(u8::try_from(id).unwrap_or_default()))
```

Comment:
this should be called with None, I believe, see `to_checksum` impl

I don't think we want the chain id prefix?

Reply:
Thanks for the review. XD I now understand what you pointed out.
Fixed in https://github.com/foundry-rs/foundry/pull/4158/commits/4bd34f8dead97d2122a0b168f970bdcb4a263230